notification/docs/DeploymentGuide.md

```diff
@@ -0,0 +1,69 @@
+# Deployment Guide
+
+<!-- START doctoc generated TOC please keep comment here to allow auto update -->
+<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
+**Table of Contents**  
+
+- [Environment variables](#environment-variables)
+  - [Adyen](#adyen)
+  - [commercetools](#commercetools)
+  - [Other Configurations](#other-configurations)
+- [Deployment](#deployment)
+  - [Docker](#docker)
+    - [Running the Docker image](#running-the-docker-image)
+  - [AWS Lambda](#aws-lambda)
+
+<!-- END doctoc generated TOC please keep comment here to allow auto update -->
+
+## Environment variables
+Following environment variables must be provided in order to run the notification module.
```

Comment:
```suggestion
Following environment variables are supported.
```

Reply:
I will keep as is, I could not get why we need to say environment variables `are supported` 🤔

Reply:
Because some variables are mandatory and some not and you state upfront that all of them are mandatory (`must`).